android/app/src/fdroid/play/listings/ar/release-notes.txt

```diff
@@ -1,12 +1,9 @@
-• New OpenStreetMap maps data as of September 20
-• New Ruler (aka Helicopter) routing for straight distance measurement
-• Background voice and directions
-• Fixed "Wait…" dialog on Android 13 with SD card
-• Do not pause music when the screen is rotated
-• Added/improved many icons and styles
-• Improved search results relevance
-• Parking entrances are searchable
-• Test TTS voice in the settings
-• Fixed arrow position button on Android 5&6
+• New OpenStreetMap data as of November 13
+• It is possible to zoom in deeper now and see more objects in dense areas
+• Outdoors map style preview (activate with ?olight or ?odark in search)
+• Fixed import of bookmarks in KMB format
+• Corrected the misleading location button's icon state
+• "Inertial" kinetic scroll improvements
+• Improved routing, search, styles, icons, and translations
```

Comment:
> "Corrected the misleading location button's icon state"
> "Improved routing, search, styles, icons, and translations" 

are too generic to understand what has changed. It would like to have these three items in the list:

- Fixed the missing 'green dot' location indicator at the right top of the Android screen
- Disabled the screen wake lock by default to fix power-saving when using the app on a locked screen
- Corrected the misleading location button's icon state when the map is moved or zoomed during a location search

Please note that the proposed text above is more specific. The user can actually validate the change and see the difference. 

> It is possible to zoom in deeper now and see more objects in dense areas

This one is also too generic. Even I don't understand what has changed. Any examples?

Reply:
Good points, but your suggested text won't fit into Android's characters limit...

Also I personally have no idea what is a "green dot location indicator". Never saw one. I have a small pin in the status bar when there is a fix. Maybe depends on android's version and/or brand.

Re "possible to zoom deeper": its about z20 addition. Now max zoom is twice more / deeper than it was before.
How would you formulate it?

Reply:
> Improved routing, search, styles, icons, and translations

"Many other style, routing, etc. improvements, see full list on omaps.org/news"

Reply:
> Also I personally have no idea what is a "green dot location indicator". 

Ok, let me rephrase.

Reply:
Hmu if you've done it and I'll start translating to 🇭🇺

Reply:
give me 10 mintues.

Reply:
• New OpenStreetMap data as of November 13
• The maximum zoom level is twice as deep as it was before.
• Fixed import of bookmarks from MAPS.ME app
• The preview of outdoor map style - activate with ?olight or ?odark in search
• Disabled the screen wake lock by default to improve power-saving
• Improved the location search indication in both the system and the app.
• "Inertial" kinetic scroll improvements
• Many other style, routing, etc. improvements, see full list on omaps.org/news


@pastk @biodranik PTAL

Reply:
As you've merged #6523 then it should be added to the release note too (perhaps instead of something else less important).

Reply:
• 2023-11-13 OpenStreetMap data
• The maximum zoom level is twice as deep as it was before
• Outdoors map style preview (activate with ?olight or ?odark in the search)
• Fixed import of bookmarks from MAPS.ME app
• Added experimental "Details on KAYAK" affiliate link to hotels
• Disabled the screen wake lock by default to improve power-saving
• Improved the location search indication in both the system and the app
• Many other style, routing, etc. improvements, see full list on omaps.org/news

Reply:
> Outdoors map style preview (activate with ?olight or ?odark in the search)

I'm afraid people will activate it and won't know how to deactivate :)

• New OpenStreetMap data as of November 13
• The maximum zoom level is twice as deep as it was before
• Fixed import of bookmarks from MAPS.ME app
• Added experimental "Details on KAYAK" affiliate link to hotels
• Disabled the screen wake lock by default to improve power-saving
• Corrected the location search indication when the map is moved or zoomed
• Outdoors map style preview - read on how to activate it and about many other improvements on omaps.org/news

480 chars

Reply:
Updated to @pastk's text.